default.js

```diff
@@ -26,29 +26,54 @@ function init_data_table() {
   // add a text input filter to each column of the new row
   $("#data thead tr:eq(1) th").each(function (i) {
     var title = $(this).text();
-    $(this).html("<input type='text' placeholder='Search '" + title + "' />");
-    $("input", this).on( "keyup change", function () {
+    $(this).html(
+      "<input type='text' class='column-filter' placeholder='Search... '" +
+        title +
+        "' />"
+    );
+    $("input", this).on("keyup change", function () {
       if (g_data_table.column(i).search() !== this.value) {
         g_data_table.column(i).search(this.value).draw();
       }
     });
   });
-  g_data_table = $('#data').DataTable({
-    "data": instances_data,
-    "rowId": 0,
-    "bPaginate": false,
-    "bInfo": false,
-    "bStateSave": true,
-    "orderCellsTop": true,
-    "fixedHeader": true,
-    "oSearch": {
-      "bRegex": true,
-      "bSmart": false
+  g_data_table = $("#data").DataTable({
+    data: instances_data,
+    rowId: 0,
+    bPaginate: false,
+    bInfo: false,
+    bStateSave: true,
+    orderCellsTop: true,
+    oSearch: {
+      bRegex: true,
+      bSmart: false,
     },
-    "aoColumnDefs": [
+    columns: [
+      { className: "" },

```

Comment:
you can also give null here. https://datatables.net/reference/option/columns.className

Reply:
@hemlata-pathak fixed